src/pytest_pystack/_monitor_process.py

```diff
@@ -23,10 +25,15 @@ def start(config):
         name="pystack_monitor",
     )
     process.start()
-    atexit.register(_stop, process, queue)
+    _STOP_LIST.append(partial(_stop, process, queue))
```

Comment:
Hm - do we actually need a list, or are you just trying to closely mimic what `atexit` does? As far as I can tell, `start()` is only ever called once (from a `pytest_configure` hook), so we should only ever need to call `_stop` once. What you've written doesn't seem incorrect, but it's a bit misleading if I'm correct that the list will never have more than 1 element. I think it would be clearer to make `process` and `queue` global variables, and have `stop` do:
```py
def stop():
    if process is not None:
        _stop(process, queue)
```